public/apisearch.js

```diff
@@ -355,6 +359,17 @@ function executeForm(pageNumber) {
 
 }
 
+if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(position => {
+        currentLatitude = position.coords.latitude;
+        currentLongitude = position.coords.longitude;
+        console.log('current Latitude: ', currentLatitude);
+        console.log('current Longitude: ', currentLongitude);
+    }, error => {
+        console.log('Need access to get location.');
+    });
+}
+
```

Comment:
This code is using the browser's location. It should be removed, and an OpenCage geo lookup used instead